linux/network/systemd.sls

```diff
@@ -0,0 +1,43 @@
+{%- from "linux/map.jinja" import network with context %}
+{%- if network.enabled and grains.get('init', None) == 'systemd' %}
+
+{%- if network.systemd is mapping %}
+{%- for config_type, configs in network.systemd.iteritems() %}
+
+{%- if config_type == 'link' %}
+/etc/udev/rules.d/80-net-setup-link.rules:
```

Comment:
because this implicit setup in conflict (it's common workaround for systemd + link rules):

```
➜  rules.d pwd
/lib/udev/rules.d
➜  rules.d cat ./73-usb-net-by-mac.rules
# Use MAC based names for network interfaces which are directly or indirectly
# on USB and have an universally administered (stable) MAC address (second bit
# is 0). Don't do this when ifnames is disabled via kernel command line or
# customizing/disabling 80-net-setup-link.rules.

IMPORT{cmdline}="net.ifnames"
ENV{net.ifnames}=="0", GOTO="usb_net_by_mac_end"

ACTION=="add", SUBSYSTEM=="net", SUBSYSTEMS=="usb", NAME=="", \
    ATTR{address}=="?[014589cd]:*", \
    TEST!="/etc/udev/rules.d/80-net-setup-link.rules", \
    IMPORT{builtin}="net_id", NAME="$env{ID_NET_NAME_MAC}"

LABEL="usb_net_by_mac_end"
```